src/openforms/js/components/admin/submissions/filter.js

```diff
@@ -0,0 +1,47 @@
+const init = async () => {
+  // Retrieve the filter link that displays only triggered rules
+  const triggeredLink = document.querySelector('.logic-rules-filter > .selected > a');
+
+  // Only triggered rules are displayed
+  document
+    .querySelectorAll('.logic-logs__col-body--trigger-' + triggeredLink.dataset.trigger)
+    .forEach(trigger => {
+      trigger.parentNode.classList.remove('logic-logs__row-body--hidden');
+    });
+};
```

Comment:
easier and more robust (since this code is duplicated here and in the click event handler) is to _use_ the click handler by simulating a click on the trigger link:

```js
triggeredLink.click()
```

Note that you must prevent crashes on other pages, so only do that if `triggeredLink` actually exists.